extensions/azurePublishNew/src/recoilModel/atoms/handOffToAdminState.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { atom } from 'recoil';
+
+import { ResourcesItem } from '../../types';
+
+export const enabledHandOffResourcesState = atom<ResourcesItem[] | undefined>({
+  key: 'handoff_optional_resources',
```

Comment:
nite: 'handoffOptionalResources' would be the more canonical key string format.

Reply:
I agree, will change it in the next push.